Zype/Helpers/AnalyticsProtocolDelegate.swift

```diff
@@ -60,4 +61,77 @@ extension PlayerVC: AnalyticsHelperProtocol {
         
         return (beacon: configUrl, customDimensions: customDimensions)
     }
+
+    func getSegmentAnalyticsFromResponse(_ playerObject: VideoObjectModel?) -> [String: Any] {
+        var customDimensions = [String: Any]()
+        customDimensions[SegmentAnalyticsAttributes.contentCmsCategory.rawValue] = "null"
+        customDimensions[SegmentAnalyticsAttributes.adType.rawValue] = "null"
+        customDimensions[SegmentAnalyticsAttributes.contentShownOnPlatform.rawValue] = "ott"
+        customDimensions[SegmentAnalyticsAttributes.streaming_device.rawValue] = "Apple AppleTV"
+        customDimensions[SegmentAnalyticsAttributes.videoAccountId.rawValue] = "416418724"
```

Comment:
This value should be inherited from the Const.swift file and not hardcoded: `static let kSegmentAccountID = "" // must have some value if kSegmentAnalytics is enabled`